app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="login-form">
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.email_field :email, autofocus: true,
+      autocomplete: "email", placeholder: "Username" %>
+    </div>
+
+    <div class="field">
+      <%= f.password_field :password, autocomplete: "current-password",
+      placeholder: "Password" %>
+    </div>
+
+    <div class="signin-btn">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
```

Comment:
- [ ] after login I encountered this error of the profile image kindly fix
![Screenshot from 2023-05-24 22-59-48](https://github.com/Balikuddembe/blog-app/assets/66097371/750152b9-dadd-4086-ba87-3992c2f0a6a4)